server/server.go

```diff
@@ -50,52 +52,56 @@ func Listen(s SFTPServer) {
 
 	listener, err := net.Listen("tcp", addr)
 	if err != nil {
-		log.Fatalf("Failed to listen on 2022: %v", err)
+		log.Printf("Failed to listen on 2022: %v\n", err)
```

Comment:
isko change krke "Failed to listen on port %i, server.port krde

Reply:
Done!